bin/corretto.bash

```diff
@@ -163,9 +163,10 @@ function download {
 
 download_github_releases 'corretto' 'corretto-8' "${TEMP_DIR}/releases-corretto-8.json"
 download_github_releases 'corretto' 'corretto-11' "${TEMP_DIR}/releases-corretto-11.json"
+download_github_releases 'corretto' 'corretto-11' "${TEMP_DIR}/releases-corretto-17.json"
```

Comment:
I guess this should be `corretto-17` as the second parameter of `download_github_releases`. 😉 

```suggestion
download_github_releases 'corretto' 'corretto-17' "${TEMP_DIR}/releases-corretto-17.json"
```

Reply:
Ha 🤦  Thanks for the fix